src/resmom_win/start_exec.c

```diff
@@ -2481,6 +2490,7 @@ start_process(task *ptask, char **argv, char **envp)
 	char			**env;
 	char			*env_str = NULL;
 	char			*p = NULL;
+	char			cmd_shell[MAX_PATH + 1] = {0};
```

Comment:
For a C compiler, a character literal is of type int. In case of C++ though, a character literal is of type char. So for C, there is actually no difference between 0 and \0.